src/components/Button.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledButton = styled.button`
```

Comment:
Would it be possible to use an anchor element actually? The only buttons on the site are all links that lead to an external source, so it would be more accessible to render them as anchors - what do you think?

Reply:
pushed!